Bottom: 58a31f684367e94964e016fa3127c338f9b99cd6
Top:    477f003dd40e5f903957a829668b41e61397811d
Author: scott snyder <sss@karma>
Date:   2019-12-30 22:03:27 -0500

agent x: missing a way to get urls of all strips.



---

diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/AgentXComics.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/AgentXComics.java
index ea65884..4416416 100755
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/AgentXComics.java
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/AgentXComics.java
@@ -12,7 +12,7 @@ public class AgentXComics extends ArchivedComic {
 
 	@Override
 	public String getComicWebPageUrl() {
-		return "http://www.agent-x.com.au/";
+		return "https://www.agent-x.com.au/";
 	}
 
 	@Override
@@ -52,7 +52,7 @@ public class AgentXComics extends ArchivedComic {
 
 	@Override
 	protected String getArchiveUrl() {
-		return "http://www.agent-x.com.au/";
+		return "https://www.agent-x.com.au/";
 	}
 
 	@Override
@@ -64,10 +64,10 @@ public class AgentXComics extends ArchivedComic {
 	protected String parse(String url, BufferedReader reader, Strip strip) throws IOException  {
 		String str;
 		String final_str = null;
-		String final_title = url.replaceAll("http.*au/comic/", "");
+		String final_title = url.replaceAll("https.*au/comic/", "");
 		final_title = "AgentX: " + final_title.replaceAll("/", "");
 		while((str = reader.readLine()) != null) {
-			int index1 = str.indexOf("webcomic-object-full");
+			int index1 = str.indexOf("current-webcomic");
 			if (index1 != -1) {
 				final_str = str;
 			}
